From 4d594929fdaa8f7231a2284e38f2ffd5ee073d33 Mon Sep 17 00:00:00 2001
From: rocketcamel <luca_lise@icloud.com>
Date: Thu, 18 Dec 2025 22:49:12 -0800
Subject: [PATCH 1/2] feat: add tea

---
 nix/modules/commonPackages.nix | 1 +
 1 file changed, 1 insertion(+)

diff --git a/nix/modules/commonPackages.nix b/nix/modules/commonPackages.nix
index 9e0a79e..a7f5f64 100644
--- a/nix/modules/commonPackages.nix
+++ b/nix/modules/commonPackages.nix
@@ -72,6 +72,7 @@
       udev
       alsa-lib
       waypipe
+      tea
     ];
     programs.nix-ld.enable = lib.mkDefault true;
     programs.zsh.enable = lib.mkDefault true;
-- 
2.51.2


From aca2c048ea755f5c8741d600781621715ee85233 Mon Sep 17 00:00:00 2001
From: rocketcamel <luca_lise@icloud.com>
Date: Thu, 18 Dec 2025 22:50:30 -0800
Subject: [PATCH 2/2] fix: update git credentials

---
 nix/users/luca/git.nix | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/nix/users/luca/git.nix b/nix/users/luca/git.nix
index 936000d..2eb06d0 100644
--- a/nix/users/luca/git.nix
+++ b/nix/users/luca/git.nix
@@ -2,7 +2,7 @@
   enable = true;
   settings = {
     user = {
-      name = "rocketcamel";
+      name = "lucalise";
       email = "luca_lise@icloud.com";
     };
     init = {
@@ -15,6 +15,6 @@
   signing = {
     signByDefault = true;
     format = "ssh";
-    key = "~/.ssh/commits.id_ed25519.pub";
+    key = "~/.ssh/id_ed25519.pub";
   };
 }
-- 
2.51.2

